scripts/build.gradle

```diff
@@ -0,0 +1,63 @@
+import java.nio.file.Files
```

Comment:
They require that they're installed before running. I could configure shellcheck to run via docker, but that is slow and requires that you're running docker. So having shellcheck installed seems less burdensome.

To complete this PR, I'd need to add instructions on how to install these tools to the README, and update the GitHub workflows to install these tools so it can be run on PRs.

Alternatively, I could check it in without adding the `check dependsOn` below. That would be an incremental step and wouldn't require all users to install black and shellcheck and wouldn't require changing the GitHub workflows. I'd still update the README with instructions on how to set up the tools and run them locally.

Reply:
We could add `shellcheck` and `black` to the `brewfile`. That would simplify stuff a lot.

Reply:
I added `shellcheck` but `black` requires `pip install` so I added a directive for that. Maybe we need a proper section on how to set up python for this project.